demo/i18n-demos.html

```diff
@@ -11,15 +11,15 @@ <h3>Internationalization</h3>
     <p>Set the <code>i18n</code> object to localize the component</p>
     <vaadin-demo-snippet id="login-demo-internationalisation">
       <template preserve-content>
-        <style include="lumo-typography lumo-color">
+        <style>
```

Comment:
this empty style block and closing `<custom-style>` tag below should be removed.

Reply:
Done